lib/api.js

```diff
@@ -119,7 +119,17 @@ async function handleCheckUpdate(context, eventName, event) {
         if (branchName != null) {
           await checkPullRequestsForBranches(context, event, branchName);
         } else {
-          logger.info("Could not find branch name in this status check result");
+          const foundPR = await checkPullRequestsForHeadSha(
+            context,
+            event.repository,
+            payload.head_sha
+          );
+          if (!foundPR) {
+            logger.info(
+              "Could not find branch name in this status check result" +
+                " or corresponding PR from a forked repository"
+            );
+          }
```

Comment:
I think this logic should all be handled in checkPullRequestsForHeadSha() directly so that the handling is symmetric to the if case (line 120)

Reply:
done